### DISCUSSION TITLE
Can i use the "Standard" Canary Channel, once i'm on Windows 11?

### DISCUSSION THREAD

**Discussion by gooloonetwork** [AveYo/MediaCreationTool.bat#269] [Q&A]
Hello. :) Can i switch to the "normal" Canary Channel, once i have Windows 11 installed or do i have to stay on the Beta Channel?

**Comment by nondetect** [comment]
Hello.  If you have compatible hardware and are using an M$ account, you can do this in the standard Insider settings.  If you don't have any of these parts, you should use a "helper" script like [ORM-Insider](https://github.com/nondetect/ORM-Insider) or similar

**Reply by gooloonetwork** [reply]
Hi, thank you. :) Another question is: Could i now just toggle of "Receive the newest Updates" on be on the "mainstream" Windows 11, so the "stable Version" or should i stay on the Channel i chose (i chose the Preview Channel)?

**Reply by nondetect** [reply]
> Hi, thank you. :) Another question is: Could i now just toggle of "Receive the newest Updates" on be on the "mainstream" Windows 11, so the "stable Version" or should i stay on the Channel i chose (i chose the Preview Channel)?

If you want receive insider build you need choose channel, "Receive the newest Update" toggle work in choosed channel, not across different channel's
